### PR TITLE
Merge grouped convolution

### PR DESCRIPTION
**Pull request by lkdci**
Regarding our discussion in https://github.com/MPolaris/onnx2tflite/issues/19, added a flag to support grouped conv with native keras Conv2d implementation instead of the split-cat workaround.
This flag should be set to true only for deployment with tflite version >= 2.9, lower versions will not support this operator and in that case the split-cat workaround should be used. The default remains to use split-cat for better support of older versions.
Some measurements I did to show the advantage of this layer:
I used the RegNetX-400MF model from `torchvision.models` which include group-conv with group size = 16.
Hardware: Galaxy S22, Android OS 12
tflite version: 2.10.0
| model | CPU backend | GPU backend |
| --- | --- | --- |
| split-cat group-conv | 7.5ms | 43.7ms |
| merged-native group-conv | 5.9ms | 7.6ms |

### CODE REVIEW

File: layers/conv_layers.py [hunk 1]
```diff
@@ -10,6 +10,17 @@
...
 LOG = logging.getLogger("convolution_layers :")
+
+
+# Whether to implement grouped convolution using the native `keras.layers.Conv2D` class with groups !=1 argument.
+# This implementation is supported only with tflite version >= 2.9.
+# If set to `False`, the grouped convolution is built using regular conv per group then concatenated as a workaround
+# to support older version of tflite.
+# Using the native keras implementation results in a simplified tflite graph and supposed to run faster.
+# See https://github.com/MPolaris/onnx2tflite/issues/19 for more details.
+USE_NATIVE_GROUP_CONV = False
```
Comment (lkdci) [inline layers/conv_layers.py:21]: I wasn't sure if it's the best place for this kind of flags. Users will only see this option if they dig down into the code. wdyt?

Comment (MPolaris) [reply layers/conv_layers.py:21]: I have no ideal for that, but this way is suitable so far. 
Maybe I will move flag to [entry function](https://github.com/MPolaris/onnx2tflite/blob/main/converter.py#L8) in the future, and I'm also a little confused about how to organize the code effectively.
Thanks for your work!

**Review by MPolaris — APPROVED** [review]
